zips/zip-0227.rst

```diff
@@ -374,6 +374,7 @@ The following is a list of rationale for different decisions made in the proposa
     - bridging information for Wrapped Assets (chain of origin, issuer name, etc)
     - information to be committed by the issuer, though not enforceable by the protocol.
 
+- We limit the size of the :math:`\mathsf{asset\_desc}` string to 512 bytes as it is a reasonable size to store metadata about the Asset, for example in JSON format. It could even be used to store small programs that could be relevant to use cases. 
```

Comment:
Not sure I understand `It could even be used to store small programs that could be relevant to use cases.`
Restructure or omit completely.

Reply:
Yeah, that came out weird. I don't think there is much benefit to the additional speculative rationale in this spec, so I'm removing it.